Tapestries/Sources/TapestryConfig/TapestryConfig.swift

```diff
@@ -0,0 +1,14 @@
+import PackageDescription
+
+let config = TapestryConfig(release: Release(actions: [.pre(.dependenciesCompatibility([.cocoapods, .spm(.all)])),
```

Comment:
Out of curiosity, why should the path be `Tapestries/Sources/TapestryConfig/TapestryConfig.swift`? Would it make sense to shorten it?

Reply:
Not really - to edit `TapestryConfig.swift` with autocompletion and all, I need a way to include `Tapestry` library; I found creating a standalone `tapestries` package away from the main project to be the least intrusive. And this standalone package is why I need to follow the SPM folder structure.

Reply:
Oh I see. In that case all good :+1:.